src/overlays/actors/ovl_En_Demo_heishi/z_en_demo_heishi.h

```diff
@@ -8,11 +8,24 @@ struct EnDemoheishi;
 typedef void (*EnDemoheishiActionFunc)(struct EnDemoheishi*, GlobalContext*);
 
 typedef struct EnDemoheishi {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x110];
-    /* 0x0254 */ EnDemoheishiActionFunc actionFunc;
-    /* 0x0258 */ char unk_258[0x68];
-} EnDemoheishi; // size = 0x2C0
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[17];
+    /* 0x1EE */ Vec3s morphTable[17];
+    /* 0x254 */ EnDemoheishiActionFunc actionFunc;
+    /* 0x258 */ s16 headRotY;
+    /* 0x25A */ s16 headRotX;
+    /* 0x25C */ s16 headRotZ;
+    /* 0x25E */ s16 headRotYTarget;
+    /* 0x260 */ s16 headRotXTarget;
+    /* 0x262 */ s16 headRotZTarget;
+    /* 0x264 */ s32 animIndex;
+    /* 0x268 */ s16 unk268;
+    /* 0x26C */ f32 frameCount;
+    /* 0x270 */ s16 unk270;
```

Comment:
I'd add a `// set and not used` here.

Reply:
Could 270 be something like "isTalking"? I noticed that it gets set to 0 in SetupIdle and 1 in SetupTalk, so it seems like a boolean of some sort.

Reply:
Possibly, but I wouldn't name it unless we find something that actually uses it. Others may be less conservative on this.

Reply:
Okay, I'll refrain from making the changing it then. The comment has also been added.